src/user-research/index.md

```diff
@@ -66,7 +66,7 @@ Our principles are not rules. They guide our work, keep us improving as a team,
 
    Researchers at dxw help our clients build on the knowledge and data they already
    have. We combine existing knowledge, poorly understood data and new research
-   into a coherent picture. And know how much it increases the credibility of our findings when we properly use and acknowledge previous work.
+   into a coherent picture. And know how much we increase the credibility of our findings when we properly use and acknowledge previous work.
```

Comment:
I think the sentence starting "And" isn't quite right -- "And then we know" or possibly even just omitting everything before "We increase"

Reply:
In a few of the other principles we have "we know x, so we do y".
Maybe:
Researchers at dxw help our clients build on the knowledge and data they already have.
We know how much we strengthen the credibility of our findings when we properly use and acknowledge previous work. So we combine existing knowledge, poorly understood data and our new research into a coherent picture.

Reply:
The last sentence is specifically about acknowledging previous research, so it's not really a follow on from the previous sentence (so I wouldn't use "then"). I think "And know" or "And we know" are okay - the former does seem a little weird, but our playbook is full of "And" sentences based on the tone of voice guidance